### DISCUSSION TITLE
PatchTool 6.51 10/31

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#400] [Announcements]
6.51相容性更新，漢化至6.4，技改數值更新至6.51。

請從release下方連結或內文連結下載檔案。以下兩種方法擇一使用即可。

## 方法一：直接覆蓋遊戲檔案

解壓縮後直接覆蓋`SquareEnix\FINAL FANTASY XIV - A Realm Reborn\game\sqpack\ffxiv`內的同名檔案即可。內含三個000000開頭的字體相關檔案與三個0a0000開頭的文本相關檔案。

設定：Windows作業系統，非高清字體，以繁體中文覆蓋日文和英文，官方渠道版本。

設定相同的人可以考慮直接下載這包，（可能）比較不會遇到問題。其他作業系統未測試。

- ffxiv-20231003.rar：請從下方連結下載。

## 方法二：使用漢化器

使用方法請參照[readme](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/2.-%E4%BD%BF%E7%94%A8)。

檔案：

- FFXIVChnTextPatch-20231031.rar：漢化器本體。需要自行調整設定的人請下載這包手動漢化。請從下方連結下載。

視需求選用：

- [font-custom-4k-smaller-20230131.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.0.1/font-custom-4k-smaller-20230131.rar)：高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)透過重複使用字體字型縮小檔案體積，使用Noto Sans。
- [font-custom-20230111.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.72.2/font-custom-20230111.rar)：舊高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)所有大小字體皆分開繪製，使用Noto Sans和Noto Serif。
- rawexd-zhcn-20231031.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [FFXIVChnTextPatch-20230603.jar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.1.1.0/FFXIVChnTextPatch-20230603.jar)：如果你有遇到JNI error，請參考 https://github.com/GpointChen/FFXIVChnTextPatch-GP/issues/90 放到與exe同位置使用。
- backup-20231031.rar：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。只適用於這個release發布時的遊戲版本。

※若不替換字體會導致遊戲內無法正常顯示部分中文字。另外，目前大廳部分中文字必須使用高畫質字體包才能正常顯示。更多資訊可參閱[wiki頁面](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v3.2.0.3'>PatchTool 6.51 10/31</a>.</em>

**Comment by jamesxiang11** [comment]
这次6.5.1的汉化不支持简体了吗？汉化界面没有语言选择，默认繁体

**Reply by GpointChen** [reply]
CSV模式可以下載簡體CSV使用（e.g. rawexd-zhcn-20231031.rar）

**Reply by GpointChen** [reply]
CSV模式下是直接讀CSV檔案，那個檔案內是簡體就會是簡體，所以沒有設定選項

**Reply by jamesxiang11** [reply]
rawexd-zhcn-20231031.rar没有链接诶，然后就是这次汉化后传送会概率dx11报错闪退

edit：刚沙都挂机直接报错闪退了

**Reply by GpointChen** [reply]
https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v3.2.0.3 最下面應該有連結

閃退問題周末我有空確認看看

**Reply by jamesxiang11** [reply]
asset里面没有找到诶

**Comment by ooo-Jay** [comment]
上一個版本hotfix更後之後, 用這版本的漢化不能完全漢化所有任務, 很多藍色任務都還是日文.
例如6.35的Deep Dungeon Eureka Orthos任務

**Reply by chun1324** [reply]
6.35更新後方法一直接覆蓋遊戲檔案已經無效

**Comment by GpointChen** [comment]
https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.2.0.3/ffxiv-20231130.rar

hotfix後使用覆蓋檔方式的玩家可以用這個頂著

漢化器版本等我把遊戲載回來後弄
